Assets/Scripts/Layers/LayerUIManager.cs

```diff
@@ -168,6 +168,8 @@ private Sprite GetProxyLayerSprite(LayerGameObject layer)
         {
             switch (layer)
             {
+                case WMSLayerGameObject _: //requested to be equal to geojsonlayer
```

Comment:
Comment heeft geen toegevoegde waarde; kan dan beter weg. En je kan dit ook oplossen door 

```
case WMSLayerGameObject _:
case GeoJsonLayerGameObject _:
    return layerTypeSprites[8];
```

Een switch statement zal zonder break of return elke case statement accepteren die bij een block code staat. Daarom zie je zo vaak bugs als mensen een break vergeten in een case :)

Reply:
nee want dan moet ik cartesiantilelayergameobject / tile3dlayergameobject ook gaan verhuizen in de switch, want wmslayergameobject is een cartesiantilelayergameobject om van de tilesysteem te kunnen gebruik maken